CODEOWNERS.md

```diff
@@ -1,15 +1,14 @@
 # Each line is a file pattern followed by one or more owners
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, @romanlutz and @riedgar-ms will be
+# requested for review when someone opens a pull request.
+* @romanlutz @riedgar-ms
 
-/devops @riedgar-ms @romanlutz
-/docs @riedgar-ms
-/fairlearn/metrics/ @riedgar-ms @rihorn2
-/fairlearn/postprocessing/ @romanlutz
+/examples/ @mmadaio @MiroDudik
```

Comment:
@hildeweerts @adrinjalali this is based on assumptions from a while ago. As the description above says this just auto-assigns people to PRs, which by no means indicates that these should be the only reviewers. Still, you may want to be auto-added to PRs from certain areas depending on your interests and/or expertise. Feel free to respond and I'll update OR directly suggest edits OR think about it and update later.

Reply:
I'm happy not to be on this file. I haven't used codeowners file before and I'm not sure how I feel about it, especially when the project is small enough to actually subscribe to it and get emails for every single event on the repo.